### PR TITLE
project-init

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

**Comment by james-simpson-ST** [comment]
Just tried to run gatsby develop and it loads the animated-card page fine but when I try the js-payments page I get the amount, name, email and phone fields but I don't see the sensitive payment fields rendered and I can't make a payment, is there something else I need to do?

**Comment by pgs-gmartowski** [comment]
> Just tried to run gatsby develop and it loads the animated-card page fine but when I try the js-payments page I get the amount, name, email and phone fields but I don't see the sensitive payment fields rendered and I can't make a payment, is there something else I need to do?

Yes, I'm currently working on this issue -  branch - `project-init`. 
I think there is a problem with animated card which is loading twice on init :/

**Comment by pgs-gmartowski** [comment]
@james-simpson-ST , @wirus15 , @msakowicz - should work fine now; it's not styled well but I want to make sure that construction of this example is ok.

File: src/pages/index.jsx [hunk 1]
```diff
@@ -1,9 +1,103 @@
...
+import SecureTrading from "../stjs/st"
+import Helmet from "react-helmet/es/Helmet"
+
 class IndexPage extends Component {
+  componentDidMount() {
+    this.loadST()
+  }
+
+  loadST() {
+    this.config = {
```
Comment (wirus15) [inline src/pages/index.jsx]: It would look nicer to move this to a separate file 😉

**Comment by wirus15** [comment]
The code looks fine to me, however iframes are not showing on my machine. This should not impact merchants though since iframe urls point to an external address. We can try to simulate this using docker but I'm not sure we need this right now :)

File: src/pages/index.jsx (hunk 1 above)
Comment (pgs-gmartowski) [reply src/pages/index.jsx]: true :) I did the same with animated card component

**Comment by pgs-gmartowski** [comment]
> The code looks fine to me, however iframes are not showing on my machine. This should not impact merchants though since iframe urls point to an external address. We can try to simulate this using docker but I'm not sure we need this right now :)

I think now this should work, what error do you receive during library init ?

**Pull request merged**